src/zerocash/Zerocash.h

```diff
@@ -33,6 +33,7 @@
 #define ZC_V_SIZE          8
 #define ZC_RHO_SIZE        32
 #define ZC_R_SIZE          48
+#define ZC_MEMO_SIZE       128
```

Comment:
(This comment is really for line 25, not changed by this diff.)

Why is ``ZC_PK_ENC_SIZE`` 311? It should be 32, right?